.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: rust
 sudo: false
-rust:
-  - nightly
-env:
-  - FEATURES=""
-  - FEATURES="unstable"
+matrix:
+  include:
+    - rust: nightly
+    - rust: nightly
+      env:
+      - FEATURES="unstable"
 script: |
-  cargo build --verbose --features "${FEATURES}"
+  cargo build --verbose --features "${FEATURES}" &&
```

Comment:
I'd actually like to leave this as-is, as it matches the [default Travis settings](http://docs.travis-ci.com/user/languages/rust/#Default-test-script).